src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritCause/description.jelly

```diff
@@ -44,7 +44,7 @@
                     <j:choose>
                         <j:when test="${other.hasBuild()}">
                             <td width="18">
-                                <img src="${imagesURL}/16x16/${other.build.iconColor.image}"/>
+                                <l:icon class="${build.build.iconColor.iconClassName}"/>
```

Comment:
Shouldn't that be `other.build.iconColor.iconClassName`?
Also in line 57 the change to `<l:icon class="icon-grey-anime icon-sm"/>` is missing similar to the change in ajaxTriggerMonitor.jelly